workspace/solution.go

```diff
@@ -0,0 +1,46 @@
+package workspace
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"time"
+)
+
+const solutionFilename = ".solution.json"
+
+// Solution contains metadata about a user's solution.
+type Solution struct {
+	Track       string    `json:"track"`
+	Exercise    string    `json:"exercise"`
+	ID          string    `json:"id"`
+	URL         string    `json:"url"`
+	Handle      string    `json:"handle"`
+	IsRequester bool      `json:"is_requester"`
+	SubmittedAt time.Time `json:"submitted_at"`
```

Comment:
Ok, I made time.Time a pointer, even though we have to jump through some hoops to take an address from a time.Time value. We'll see how painful it is and can always change it later if necessary.